lib/src/core/multiquery_client.dart

```diff
@@ -0,0 +1,79 @@
+part of 'package:web3dart/web3dart.dart';
+
+class MultiQueryWeb3Client extends Web3Client {
+  MultiQueryWeb3Client(
+    String url,
+    Client httpClient, {
+    SocketConnector? socketConnector,
+  }) : super.custom(
+          JsonRPCMultiQuery(url, httpClient),
+          socketConnector: socketConnector,
+        );
+
+  /// Method used to make several contract calls and/or various rpc queries, using only
+  /// one request to the eth client.
+  /// The resulting list of responses will a mix of [RPCError] instance/s and/or
+  /// [EthRPCQuery] instance/s with the returned value
+  Future<List<dynamic>> multiqueryCall(
```

Comment:
method name should be camelCase.

Reply:
it was only meant for people who actually want to make a multiqueryCall.
Plus i didnt want to mess with the Web3Client class, this class only adds this functionality.
It can be done but its more code in one file, and it doesnt suit all the use cases for people using this lib.
though i believe documentation regarding the use of this class should be added

Reply:
I agree with docs :)